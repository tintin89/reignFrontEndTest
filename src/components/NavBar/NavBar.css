.navBar{
    height: 230px;
    width: 100%;
    display: flex;
    flex-flow: column;
    }

  

.nav{
    flex: 0.4;
    display: flex;
    justify-content: center;
    align-items: flex-end;    
}    

.nav ul{
    list-style: none;
    display: flex;
}
.nav ul li{
     /* 
    cursor: pointer;
    width: 98px;
    height: 31px;
    padding: 0.5rem 2.5rem;
    border-radius: 2px;    
    font-family: Roboto;
    font-size: 16px;
    font-weight: 500;
    font-stretch: normal;
    font-style: normal;
    line-height: 1.75;
    letter-spacing: normal;
    text-align: center;       
    border: solid 1px #d6d6d6;
    */
}

.active{
    border: solid 1px #1797ff;
    color: #1797ff;
}
a{
    color: #6a6a6a;
    text-decoration: none;
    cursor: pointer;
    width: 98px;
    height: 31px;
    padding: 0.5rem 2.5rem;
    border-radius: 2px;    
    font-family: Roboto;
    font-size: 16px;
    font-weight: 500;
    font-stretch: normal;
    font-style: normal;
    line-height: 1.75;
    letter-spacing: normal;
    text-align: center;       
    border: solid 1px #d6d6d6;
    transition: ease-out 0.3s;   
}
 

