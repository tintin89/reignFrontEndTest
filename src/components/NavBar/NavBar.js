import React from 'react';
import './NavBar.css';
import TitleBar from './TitleBar/TitleBar';

function NavBar() {
    return (
        <header className="navBar">
            <TitleBar/>
            <nav>
                <ul>
                    <li>All</li>
                    <li>My faves</li>
                </ul>
            </nav>
        </header>
    )
}

export default NavBar
