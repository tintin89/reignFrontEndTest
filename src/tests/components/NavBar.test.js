import React from 'react';
import  { shallow}  from "enzyme";
import NavBar from '../../components/NavBar/NavBar';


describe('tests in <NavBar/>',()=>{   
  

    test('Must show correctly the component',()=>{       
        const wrapper = shallow(<NavBar/>);
        expect(wrapper).toMatchSnapshot()
    }) 
   
})