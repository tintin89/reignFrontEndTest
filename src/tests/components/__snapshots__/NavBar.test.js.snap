// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in <NavBar/> Must show correctly the component 1`] = `
<header
  className="navBar"
>
  <TitleBar />
  <div
    className="nav"
  >
    <ul>
      <li>
        <NavLink
          exact={true}
          to="/"
        >
          All
        </NavLink>
      </li>
      <li>
        <NavLink
          to="/faves"
        >
          My faves
        </NavLink>
      </li>
    </ul>
  </div>
</header>
`;
